programs/stake/src/stake_state.rs

```diff
@@ -281,17 +316,17 @@ fn calculate_stake_rewards(
     vote_state: &VoteState,
     stake_history: Option<&StakeHistory>,
     inflation_point_calc_tracer: Option<impl Fn(&InflationPointCalculationEvent)>,
-    _fix_activating_credits_observed: bool, // this unused flag will soon be justified by an upcoming feature pr
+    credits_auto_rewind: bool,
 ) -> Option<(u64, u64, u64)> {
```

Comment:
Thoughts of making this return type an `Option<struct>` as well, while you're in here? (I have to keep scrolling all over the place to remember what the values are)

Reply:
sorry but I'll pass on.

Reply:
😢